test/cmd/newapp.sh

```diff
@@ -469,6 +469,12 @@ os::cmd::expect_success 'oc new-app openshift/bogusimage https://github.com/open
 
 os::cmd::expect_success 'oc create -f test/testdata/installable-stream.yaml'
 
+# ensure that --strategy sets the build strategy
+os::cmd::expect_success_and_text 'oc new-build --name sourcetest python~https://github.com/sclorg/django-ex --source-image centos:latest --source-image-path /tmp --strategy source --dry-run -o yaml' 'sourceStrategy'
+os::cmd::expect_success_and_text 'oc new-build --name sourcetest python~https://github.com/sclorg/django-ex --source-image centos:latest --source-image-path /tmp --strategy pipeline --dry-run -o yaml' 'jenkinsPipelineStrategy'
+os::cmd::expect_success_and_text 'oc new-build --name sourcetest python~https://github.com/sclorg/django-ex --source-image centos:latest --source-image-path /tmp --strategy docker --dry-run -o yaml' 'dockerStrategy'
```

Comment:
can you also add a test that confirms you can create a build with just image inputs (w/o passing the --binary flag)?

Reply:
updated